src/utils.js

```diff
@@ -16,7 +16,12 @@ export function getScrollbarWidth() {
 }
 
 export function setScrollbarWidth(padding) {
-  document.body.style.paddingRight = padding > 0 ? `${padding}px` : null;
+  const paddingRight = padding > 0 ? `${padding}px` : null;
+  const fullWidthFixedContent = document.querySelectorAll('.fixed-top, .fixed-bottom');
+  const elementsAffectedByScrollbarWidth = [ document.body, ...fullWidthFixedContent ];
+  elementsAffectedByScrollbarWidth.forEach(element => {
+    element.style.paddingRight = paddingRight;
```

Comment:
When these component re-render, it will end up removing the padding which is added here.

Reply:
Good catch. Back to the drawing board.

Reply:
The only why I can think this would work would be using context so that a provider can inform the consumers what the padding currently is as well as re-render them when it changes.